### PR TITLE
Fix comments count code + fix all platforms with evolution script

### LINKED ISSUE #2194: rate count on meinberlin includes all versions of edited rates

### PR DESCRIPTION
**Pull request by pallix**
Fixes #2194  and #2188

### CODE REVIEW

**Comment by pallix** [comment]
@slomo can you have a look?

**Comment by slomo** [comment]
Should be reviewed by someone with more backend experience.

File: src/adhocracy_core/adhocracy_core/evolution/__init__.py [hunk 1]
```diff
@@ -630,6 +630,37 @@ def add_image_reference_to_proposals(root):  # pragma: no cover
...
+                                  resolve=True,
+                                  indexes={'tag': 'FIRST'})
+    comment_versions = catalogs.search(query).elements
+    count = len(comment_versions)
+    for index, comment in enumerate(comment_versions):
+        logger.info('Recalculate comment_count for resource {0} of {1}'
+                    .format(index + 1, count))
+        update_comments_count(comment, 1, registry)
+
+
```
Comment (joka) [inline src/adhocracy_core/adhocracy_core/evolution/__init__.py]: why did you not fix set_comment_count? Or we should add a Note that set_comments_count is broken and should not be used

Comment (pallix) [inline src/adhocracy_core/adhocracy_core/evolution/__init__.py]: I didn't fix it to ease deployment: we don't need to mark some evolution steps unfinished in Salt just to remove them after and to track for which platform it was done.

**Pull request merged**